workers-docs/src/content/tooling/wrangler/commands.md

```diff
@@ -1,268 +1,233 @@
 ---
 title: Commands
-weight: 2
+weight: 1
 ---
 
 - [👯 `generate`](#generate)
 - [📥 `init`](#init)
 - [🦀⚙️ `build`](#build)
 - [🔧 `config`](#config)
 - [☁️ 🆙 `publish`](#publish)
-  * [workers.dev](#publishing-to-workersdev)
-  * [Registered domain](#publishing-to-your-own-domain)
-  * [Multiple domains](#publishing-the-same-code-to-multiple-places)
+  - [workers.dev](#publishing-to-workersdev)
+  - [Registered domain](#publishing-to-your-own-domain)
+  - [Multiple domains](#publishing-the-same-code-to-multiple-places)
 - [👂 `dev`](#dev-alpha-)
 - [🦚 `tail`](#tail)
 - [🔬 `preview`](#preview)
-  * [Making it work with WSL](#making-preview-work-with-wsl2)
+  - [Making it work with WSL](#making-preview-work-with-wsl2)
 - [🗂️ `kv`](#kv)
 - [🤫 `secret`](#secret)
 
+### generate
 
-###  generate
+Scaffold a project, including boilerplate for a Rust library and a Cloudflare Worker.
+You can pass a name and template to this command optionally.
 
-  Scaffold a project, including boilerplate for a Rust library and a Cloudflare Worker.
-  You can pass a name and template to this command optionally.
-
-  ```bash
-  wrangler generate <name> <template> --type=["webpack", "javascript", "rust"] --site
-  ```
-
-  All of the arguments and flags to this command are optional:
-
-  - `name`: defaults to `worker`
-  - `template`: defaults to the [`https://github.com/cloudflare/worker-template`](https://github.com/cloudflare/worker-template)
-  - `type`: defaults to ["webpack"](/tooling/wrangler/webpack)
-  - `--site`: generates a [Workers Site](/sites) from an existing static site
+```bash
+wrangler generate [$NAME] [$TEMPLATE] [--type=${"webpack" | "javascript" | "rust"}] [--site]
+```
 
-###  init
+|             |                                                              | Optional | Default value                                                |
+| ----------- | ------------------------------------------------------------ | -------- | ------------------------------------------------------------ |
+| `$NAME`     | Name of Worker                                               | Optional | "worker"                                                     |
+| `$TEMPLATE` | Github URL of the template to base new project off of        | Optional | [`worker-template`](https://github.com/cloudflare/worker-template) |
+| `--type`    | Type of project                                              | Optional | "webpack"                                                    |
```

Comment:
well, we have `webpack` documented [here](https://developers.cloudflare.com/workers/tooling/wrangler/webpack) but idk about the other types. we should probably document somewhere what they each do if we dont already.

`javascript` does nothing + looks for script defined in package.json "main" field
`rust` uses `wasm-pack` to build your worker - elaborate more. idk if we even document this anywhere, might be worth a separate issue since this PR is already big

Reply:
Yeah seems that is adding content I created a separate issue https://github.com/cloudflare/workers-docs/issues/898